LICENSE.md

```diff
@@ -6,14 +6,14 @@ Crediting Who's On First is recommended and linking back to the License is requi
 
 The Who's On First dataset is both an original work and a modification of existing open data (sources listed below). Some of those open data projects do require attribution. We detail all sources and their specific license, usage, and vintage metadata in the full [sources list](https://whosonfirst.org/docs/sources/list/).
 
-_This file was generated by the `bin/docs.py` script on 2020-10-08._
+_This file was generated by the `bin/docs.py` script on 2021-03-08._
 
 - **Austria Open Data**: source [data](https://www.data.gv.at/katalog/dataset/c33d36b0-f184-4f2a-89cc-839ca7fcf88a) and [license](http://creativecommons.org/licenses/by/3.0/at/)
 - **Ordnance Survey of Northern Ireland**: source [data](https://www.opendatani.gov.uk/) and [license](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
 - **Alameda County Data Sharing Initiative**: source [data](https://data.acgov.org/) and [license](https://data.acgov.org/terms-of-use)
 - **Acme Laboratories**: source [data](https://acme.com/) and [license](https://acme.com/license.html)
 - **Amsterdam Open Datakaart**: source [data](https://kaart.amsterdam.nl) and [license](https://kaart.amsterdam.nl/datasets)
-- **Ciudad Autónoma de Buenos Aires, Iniciativa de Datos Públicos y Transparencia**: source [data](https://data.buenosaires.gob.ar/dataset/barrios) and [license](https://data.buenosaires.gob.ar/tyc)
+- **Ciudad Aut?noma de Buenos Aires, Iniciativa de Datos P?blicos y Transparencia**: source [data](https://data.buenosaires.gob.ar/dataset/barrios) and [license](https://data.buenosaires.gob.ar/tyc)
```

Comment:
Hmm.. I thought I found a fix for some Unicode errors I'm hitting when running the Makefile, but apparently that fix didn't work as expected. 

Rather than add additional dependencies to fix any Unicode issues, I'm just going to remove the handful of Unicode errors from these three sources to simplify things. 

cc @nvkelso any issue with me updating these sources as part of this PR, too?

Reply:
That's fine.